src/Tribe/Ignored_Events.php

```diff
@@ -975,7 +977,33 @@ public function hook() {
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 
+			add_filter( 'wp_count_posts', [ $this, 'patch_count_posts' ], 10 );
```

Comment:
Since you're using the default priority, you can remove it.

Reply:
```suggestion
			add_filter( 'wp_count_posts', [ $this, 'patch_count_posts' ] );
```